src/main/java/com/puppycrawl/tools/checkstyle/utils/TokenUtil.java

```diff
@@ -269,4 +269,23 @@ public static boolean areOnSameLine(DetailAST ast1, DetailAST ast2) {
         return ast1.getLineNo() == ast2.getLineNo();
     }
 
+    /**
+     * Determines if the AST belongs to the given types.
+     *
+     * @param type the Token Type to check
+     * @param types the acceptable types
+     *
+     * @return true if type matches one of the given types.
+     */
+    public static boolean isOfType(int type, int... types) {
+        boolean result = false;
+        for (int t : types) {
```

Comment:
From https://github.com/checkstyle/checkstyle/issues/8048#issuecomment-618410435: I still think that a declarative programming approach with streams is a much more elegant implementation than this.

Reply:
I think you're right. That's a better implementation.

@romani @rnveach ping

Reply:
totally agree with @wltan

Reply:
Done!